client/src/components/AddNewProfile.js

```diff
@@ -96,187 +86,202 @@ const AddNewProfile = () => {
 
   const nationalityOptions = nationalities.map(nationality => ({
     key: nationality.id,
-    text: nationality.nationality,
+    label: nationality.nationality,
     value: nationality.id
   }));
 
   const groupsOptions = groups.map(group => ({
     key: group.id,
-    text: group.group_name,
+    label: group.group_name,
     value: group.id
   }));
 
   return (
-    <Segment style={{ border: " 3px solid #d1a1b7" }}>
-      <Form onSubmit={createProfile}>
-        <ul class="errors">
+    <div className="px-10 max-w-4xl mx-auto">
+      <form onSubmit={createProfile}>
```

Comment:
if there is too much forms and validation work in the app, then I would look at using [formik](https://formik.org/) and [yup](https://github.com/jquense/yup)

Reply:
Thanks for the suggestions. The current validation her isn't really adequate so it'll be a task to improve it.